ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py

```diff
@@ -70,7 +70,6 @@ def tan(
         ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
     return ret
 
-
```

Comment:
You should not remove this blank line as the formatting rule requires two blank line after function definitions.